src/zone/worker-context.h

```diff
@@ -33,6 +33,9 @@ namespace zone {
         /// <summary> UV Event loop. </summary>
         EVENT_LOOP,
```

Comment:
Seems we don't need EVENT_LOOP in tls, right?

Reply:
We'd better remove it from tls to eliminate the chance of uv operation directly on the loop by later developers.